app/Models/Barangay.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Barangay extends Model
+{
```

Comment:
Put fillables column name here even though we are not using it. To be able to track database table easily on our model.